util/auth.js

```diff
@@ -0,0 +1,21 @@
+import jwt from 'jsonwebtoken';
+import bcrypt from 'bcrypt';
+import secret from '../config/config';
```

Comment:
Secrets should be stored in .env and pulled from there using **process.env.SECRET_NAME**